test/utils/TestConstants.sol

```diff
@@ -1,4 +1,4 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.25;
 
-uint8 constant TEST_DEFAULT_OWNER_FUNCTION_ID = 0;
+uint32 constant TEST_DEFAULT_OWNER_FUNCTION_ID = 0;
```

Comment:
Might want to rename to  `TEST_DEFAULT_OWNER_ENTITY_ID`

Reply:
This will be address in 4/n